gilbreth_perception/config/model_list.yaml

```diff
@@ -0,0 +1,15 @@
+part_list:
+ - name: disk 
+   path: /model/disk.pcd
```

Comment:
It is relative to the path of gilbreth_perception package.
In my code, the full_path= package_path+ model_path.
package_path was set in roslaunch file by: <param name="package_path" value="$(find gilbreth_perception)"/>